client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1523,12 +1524,11 @@ Undocumented.prototype.fetchDns = function( domainName, fn ) {
 	this.wpcom.req.get( '/domains/' + domainName + '/dns', fn );
 };
 
-Undocumented.prototype.addDns = function( domain, record, fn ) {
```

Comment:
Aaaargh, your right, I confused actions with endpoints :persevere: Sorry about that.